code/game/objects/structures.dm

```diff
@@ -147,6 +147,14 @@
 
 	user.forceMove(TT)
 
+	var/list/grabbed_things = list()
+	for(var/obj/item/grab/grab in list(user.l_hand, user.r_hand))
+		grabbed_things += grab.grabbed_thing
+		grab.grabbed_thing.forceMove(user.loc)
+	user.forceMove(TT)
```

Comment:
I may be missing something but why is this forceMove necessary? It seems like it's doubled with the one above.